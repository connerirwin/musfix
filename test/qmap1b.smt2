; Constraints can generate multiple variants
(qualif Sum ((a Int)(b Int)(c Int)) (= (+ a b) c))

(wf $k0 ((v0 Int)(v1 Int)(v2 Int)))
