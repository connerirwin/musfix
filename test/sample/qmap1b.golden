
Preparing to find greatest fixpoint...

Inputs
--------
Reading from file test/sample/qmap1b.msmt
Qualifier "Sum" [Var IntS "a",Var IntS "b",Var IntS "c"] (Binary Eq (Binary Plus (Var IntS "a") (Var IntS "b")) (Var IntS "c"))
WFConstraint "$k0" [Var IntS "v0",Var IntS "v1",Var IntS "v2"]

QMAP
--------
fromList [("$k0",QSpace {_qualifiers = [Binary Eq (Binary Plus (Var IntS "v0") (Var IntS "v1")) (Var IntS "v2"),Binary Eq (Binary Plus (Var IntS "v0") (Var IntS "v2")) (Var IntS "v1"),Binary Eq (Binary Plus (Var IntS "v1") (Var IntS "v0")) (Var IntS "v2"),Binary Eq (Binary Plus (Var IntS "v1") (Var IntS "v2")) (Var IntS "v0"),Binary Eq (Binary Plus (Var IntS "v2") (Var IntS "v0")) (Var IntS "v1"),Binary Eq (Binary Plus (Var IntS "v2") (Var IntS "v1")) (Var IntS "v0")], _maxCount = 6})]

Candidates
--------
[Candidate {solution = fromList [("$k0",fromList [])], validConstraints = fromList [], invalidConstraints = fromList [], label = "0"}]


Final candidates: 
0: [$k0 -> {}] (0 0)

